modulefiles/analyses/analyses_plots.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
-# modulefile for EVS subseasonal component
+# modulefile for EVS analyses plot component
 
-set +x
+set -x
 
-module use /apps/prod/lmodules/intel/${intel_ver}
+module load PrgEnv-intel/${PrgEnvintel_ver}
+module load intel/${intel_ver)
```

Comment:
Please correct this typo. Needs a } instead of ).

Reply:
Fixed.